bears/mp3/MP3CheckBear.py

```diff
@@ -0,0 +1,23 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='mp3chek',
+        output_format='regex',
+        output_regex=r'L(?P<line>\d+)C(?P<column>\d+): (?P<message>.*)')
+class MP3CheckBear:
+    """
+    Report possible security weaknesses for MP3 files.
+    For more information,
+    consult <https://code.google.com/archive/p/mp3check/>.
+    """
+
+    REQUIREMENTS = {DistributionRequirement(apt_get='mp3check')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        pass
```

Comment:
I think its better to discuss over here. Just copying comments from the issue:
've created basic skeleton, and will try to add more options that are available in mp3check. It might take some time as there are lot of options available with mp3check.
Just to clear some doubts:
For testing this bear, which file should I be using ? I mean, to commit the file in tests/mp3/test_files/<file_name>.mp3 as open source, I need a file that is free and open source already, right ?